File: src/main/java/com/skuniv/bigdata/domain/dto/open_api/CharterWithRentBodyDto.java
package com.skuniv.bigdata.domain.dto.open_api;

import lombok.*;

import javax.xml.bind.annotation.*;
import java.util.List;

@Getter
@Setter
@ToString
@NoArgsConstructor
@XmlRootElement(name = "body")
@XmlAccessorType(XmlAccessType.FIELD)
public class CharterWithRentBodyDto {
    @XmlElementWrapper(name = "items")
    @XmlElement(name = "item")
    private List<CharterWithRentItemDto> item;

    @XmlElement(name = "numOfRows")
    private int numOfRows;

    @XmlElement(name = "pageNo")
    private int pageNo;

    @XmlElement(name = "totalCount")
    private int totalCount;
}

File: src/main/java/com/skuniv/bigdata/domain/dto/open_api/CharterWithRentBodyDto.kt
package com.skuniv.bigdata.domain.dto.open_api

import javax.xml.bind.annotation.*

@XmlRootElement(name = "body")
@XmlAccessorType(XmlAccessType.FIELD)
data class CharterWithRentBodyDto(

        @XmlElementWrapper(name = "items")
        @XmlElement(name = "item")
        var item: List<CharterWithRentItemDto>? = null,

        @XmlElement(name = "numOfRows")
        var numOfRows: Int = 0,

        @XmlElement(name = "pageNo")
        var pageNo: Int = 0,

        @XmlElement(name = "totalCount")
        var totalCount: Int = 0
)